fiftyone/server/query.py

```diff
@@ -196,7 +197,7 @@ def modifier(doc: dict) -> dict:
     async def resolver(
         cls, name: str, view: t.Optional[BSONArray], info: Info
     ) -> t.Optional["Dataset"]:
-        return await serialize_dataset(name, view)
+        return await serialize_dataset(urllib.parse.unquote(name), view)
```

Comment:
It "should" be, but it's not currently true. If I remove this change, selecting a dataset with encodable chars like spaces from the dropdown fails to resolve because the name used to load a dataset will be the encoded string.
<img width="1185" alt="image" src="https://user-images.githubusercontent.com/30936736/202257563-059276d7-79fd-4d87-a0b8-c390edc7c7ce.png">
```
[dev:py] Traceback (most recent call last):
[dev:py]   File "/Users/kacey/.pyenv/versions/3.9.14/envs/fo/lib/python3.9/site-packages/graphql/execution/execute.py", line 528, in await_result
[dev:py]     return_type, field_nodes, info, path, await result
[dev:py]   File "/Users/kacey/.pyenv/versions/3.9.14/envs/fo/lib/python3.9/site-packages/strawberry/schema/schema_converter.py", line 513, in _async_resolver
[dev:py]     return await await_maybe(_get_result(_source, strawberry_info, **kwargs))
[dev:py]   File "/Users/kacey/.pyenv/versions/3.9.14/envs/fo/lib/python3.9/site-packages/strawberry/utils/await_maybe.py", line 12, in await_maybe
[dev:py]     return await value
[dev:py]   File "/Users/kacey/Dev/voxel51/fiftyone/fiftyone/server/query.py", line 202, in resolver
[dev:py]     return await serialize_dataset(name, view)
```
This can be removed once we have the url_name/slug implemented.